CODEOWNERS

```diff
@@ -1 +1 @@
-* @markmandel @xampprocky @luna-duclos @rezvaneh @vallfors @bombsimon
+* @markmandel @xampprocky @No-ops @rezvaneh @vallfors
```

Comment:
Noticed this message:

> Unknown owner on line 1: make sure @No-ops exists and has write access to the repository 

I'm assuming I should add them to the repository? Let me know, and merge when ready. 👍🏻

Reply:
@markmandel Yes please 🙂

Reply:
Invite incoming!